src/main/java/org/apache/commons/imaging/formats/png/PngImageParser.java

```diff
@@ -254,9 +254,8 @@ public void readSignature(final InputStream is) throws ImageReadException,
         }
 
         final PngChunkIccp pngChunkiCCP = (PngChunkIccp) chunks.get(0);
-        final byte[] bytes = pngChunkiCCP.getUncompressedProfile(); // TODO should this be a clone?
 
-        return (bytes);
+        return (pngChunkiCCP.getUncompressedProfile());
```

Comment:
This removes the `TODO` comment that was left there by Sebb during one of his reviews. Better leave it there until someone else has time to ponder on that item, and maybe remove the redundant variable.

Reply:
Reverted
TY